apps/namadillo/src/App/AccountOverview/NamBalanceContainer.tsx

```diff
@@ -102,66 +108,106 @@ export const NamBalanceContainer = (): JSX.Element => {
     shieldedAmountQuery,
     shieldedNamAmount,
   } = useBalances();
-
+  const isUnbondingOrWithdrawable =
+    unbondedAmount.gt(0) || withdrawableAmount.gt(0);
   return (
     <>
       {maspEnabled && (
         <Heading className="text-sm mb-4" level="h3">
           NAM Balance
         </Heading>
       )}
-      <div className="flex gap-2 w-full">
+      <div className="flex flex-col md:flex-row gap-2 w-full">
         <AtomErrorBoundary
           result={[balanceQuery, stakeQuery]}
           niceError="Unable to load balances"
         >
-          <Stack
-            as="ul"
-            gap={2}
-            className={clsx("w-full", {
-              "order-1": maspEnabled && shieldingRewardsEnabled,
-            })}
-          >
-            <NamBalanceListItem
-              title="Transparent NAM"
-              color={colors.balance}
-              amount={availableAmount}
-              isLoading={isLoading}
-              isSyncing={balanceQuery.isFetching}
-            />
-            <NamBalanceListItem
-              title="Staked NAM"
-              color={colors.bond}
-              amount={bondedAmount}
-              isLoading={isLoading}
-              isSyncing={stakeQuery.isFetching}
-            />
-            <NamBalanceListItem
-              title="Unbonded NAM"
-              color={colors.unbond}
-              amount={unbondedAmount.plus(withdrawableAmount)}
-              isLoading={isLoading}
-              isSyncing={stakeQuery.isFetching}
-            />
-          </Stack>
-          <Stack className="w-full" gap={2}>
-            <NamBalanceListItem
-              title="Shielded NAM"
-              color={colors.shielded}
-              amount={maspEnabled ? shieldedNamAmount : new BigNumber(0)}
-              isLoading={shieldedAmountQuery.isLoading}
-              isSyncing={isShieldSyncing}
-              isEnabled={maspEnabled}
-            />
-            <ListItemContainer
-              isEnabled={shieldingRewardsEnabled}
-              className="flex flex-1"
+          <>
+            <Stack
+              as="ul"
+              gap={2}
+              className={clsx("w-full", {
+                "order-1": maspEnabled && shieldingRewardsEnabled,
+              })}
             >
-              <ShieldedRewardsBox
-                shieldedRewardsAmount={shieldedRewards.amount}
+              <NamBalanceListItem
+                className={clsx({ "h-full": !isUnbondingOrWithdrawable })}
```

Comment:
I believe we don't need these class as a conditional, we could simply always apply it 

https://github.com/user-attachments/assets/ba57b49c-2402-4879-82d0-71b5f88d81ed

Reply:
Nice one!